src/main.py

```diff
@@ -468,6 +527,24 @@ def analyze_line(line: str) -> str:
     elif intim_pat.match(line):
         user = identify_player(line, intim_pat)
         converted = f'|-ability|p{user+1}a: {players[user].currentmon.nick}|Intimidate|boost'
+        # this has to be awful code but it works
+        l = line_num + 1
+        while 1:
+            next_line = analyze_line(log_arr[l])
+            # we might encounter a message unrelated to the game (only chat?) and we have to skip it
+            if "|c|" in next_line:
+                l += 1
+            # or a drop message in which case its not clear body
+            elif "-unboost" in next_line:
+                break
+            #otherwise assume it's clear body?
```

Comment:
Don't really need this check, just do `"-unboost" not in next_line`

Reply:
if we change it to ``if "|c|" in line:
...
elif "-unboost" not in line: `` we still need an ``else: break`` to break out of the while loop in case the next line returns a drop message, i don't think inverting the condition helps with anything